shared/src/main/scala/mouse/string.scala

```diff
@@ -65,4 +65,18 @@ final class StringOps(private val s: String) extends AnyVal {
 
   private def parse[A](f: String => A): NumberFormatException Either A = Either.catchOnly[NumberFormatException](f(s))
 
+  /**
+   * Wraps a `String` in `Throwable`.
+   */
+  @inline def asThrowable: Throwable = new Throwable(s)
```

Comment:
What is the use case for a Throwable that isn't either an Error or an Exception?  I guess it can avoid widening in invariant contexts.

Reply:
Also for the indecisive I guess, lol. It's a pretty common pattern - most people just want error messages. Small projects are just not worth the effort to think about custom error ADTs let alone the errors vs exceptions conundrum.

Not that it's a good practice in terms of Java error handling convention, but so are non-specific errors/exceptions.

Reply:
Yeah, I still think in terms of the Java hierarchy, even having not written much Java in the last 10 years.  I'm pretty well convinced by having the unspecific type just for invariant contexts, so I think this is good as is.